### ISSUE TITLE
Degree of establishment mapping

### ISSUE THREAD

**Issue by LienReyserhove** [trias-project/alien-mollusca-checklist#10]
The degree of establishment for a species is already mapped to the [Blackburn et al. 2011](http://doc.rero.ch/record/24725/files/bach_puf.pdf) vocabulary. This is an example of the content:

|degree_of_establishment | records|
|:-----------------------|-------:|
|B2                      |       5|
|B3                      |       1|
|C0                      |       6|
|C0 - C3                 |       1|
|C1 - C3                 |       1|
|C3                      |       1|
|C3, D1, D2              |       1|
|D2, E                   |       1|
|E                       |      12|

A few species are categorized under more then one category. My first reaction was to map each catogory per species as a separate row, but this might give a wrong impression when interpreting the data. I think that, by including different categories, the author means that we are not certain about the exact degree of establishment. Therefore I think we should map the categories together:

|degree_of_establishment | mapping|
|:-----------------------|-------:|
|C0 - C3                 |       C0, C1, C2, C3|
|C1 - C3                 |       C1, C2, C3|
|C3, D1, D2              |       C3, D1, D2|

@peterdesmet?

**Comment by peterdesmet** [comment]
I think @timadriaens @qgroom should decide this:

If a taxon is mapped to a range or multiple Blackburn `degree_of_establishment` values, how should that be expressed in Darwin Core: have one row for each `degree_of_establishment` listed?

**Comment by qgroom** [comment]
Perhaps you have a specific case, but in my head there should only every be one for a specific time period and area.

**Comment by qgroom** [comment]
If within a time period and area there are both temporary and established populations, then I think we would default to the most established of Blackburn's categories.

**Comment by timadriaens** [comment]
I agree, the framework is more or less hierarchical in the sense that a species "achieves" a certain degree of establishment and that is also what is relevant for policy and management
